tests/output/ConsoleInteractionTest.py

```diff
@@ -849,3 +849,16 @@ def test_empty_list(self):
                                 None,
                                 None,
                                 None)
+
+    def test_formatted_string(self):
```

Comment:
I think you should rename this test to test_source_lines to make it more clear it is testing the functionality you added.